docs/operators/rest.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "@rest"
+description: Exposing REST endpoints on top of GraphQL via @rest directive.
+---
+
+Every one needs orchestration for APIs but everyone can't move to GraphQL and there are valid reasons for not moving to GraphQL. This feature allows people to building the Tailcall DSL, leveraging the GraphQL capabilities at compile time and generating REST endpoint for actual use instead. REST is a well-known standard so works with traditional API Infrastructure - CDNs Gateways etc.
+
+This guide will show you how to use the `@rest` directive to create a RESTful service over a GraphQL layer
+
+## The @rest Directive Basics
+
+- **method**: Specifies the HTTP method (GET, POST, etc.).
+
+- **path**: Defines the endpoint URL, supporting dynamic values from query arguments.
+
+## Example: Retrieving Product Information
+
+![Diagram](/images/docs/rest-diagram.svg)
```

Comment:
Image still talks about products however the example talk about /posts and /users.